pkg/injection/codemodule/installer/image/unpack.go

```diff
@@ -75,10 +74,7 @@ func (installer *Installer) pullOCIimage(image containerv1.Image, imageName stri
 		return errors.WithStack(err)
 	}
 
-	// ref.String() is consistent with what the user gave, ref.Name() could add some prefix depending on the situation.
-	// It doesn't really matter here as it's only for a temporary dir, but it's still better be consistent.
-	imageCachePath := path.Join(imageCacheDir, base64.StdEncoding.EncodeToString([]byte(ref.String())))
-	if err := crane.SaveOCI(image, imageCachePath); err != nil {
+	if err := crane.SaveOCI(image, path.Join(imageCacheDir, ref.Identifier())); err != nil {
```

Comment:
another possible mistake

Reply:
do you think we should revert to original code `crane.SaveOCI(image, imageCachePath)`?

Reply:
fxied in 3922d71b